### PR TITLE
fix: save chart as flow

### PR DESCRIPTION
**Pull request by rephus**
Closes: [#6511](https://github.com/lightdash/lightdash/issues/6511)

### Description:

[Screencast from 07-08-23 11:04:12.webm](https://github.com/lightdash/lightdash/assets/1983672/45cf3354-fd01-4e00-8914-906aaf607b2f)

### CODE REVIEW

File: packages/frontend/src/components/common/modal/ChartCreateModal.tsx [hunk 1]
```diff
@@ -98,7 +99,7 @@ const ChartCreateModal: FC<ChartCreateModalProps> = ({
               })
             : undefined;
 
-        const savedQuery = mutateAsync({
+        const savedQuery = await mutateAsync({
```
Comment (joaoviana) [inline packages/frontend/src/components/common/modal/ChartCreateModal.tsx:102]: 🙌

**Review by joaoviana — APPROVED** [review]

**Comment by joaoviana** [comment]
will just do one last check on Render once available 👍

**Comment by rephus** [comment]
Retriggered render

**Comment by joaoviana** [comment]
looks good @rephus !

**Pull request merged**